src/widgets/modals/SearchForm.js

```diff
@@ -37,8 +41,7 @@ const SearchListItemColumn = ({ value, type }) => (
   </View>
 );
 
-const SearchListItem = ({ listItem, listConfig }) => {
-  const onPress = item => console.log(item);
+const SearchListItem = ({ listItem, listConfig, onPress }) => {
```

Comment:
should probs memo this with `React.memo` and the column

Reply:
Maybe incorrect, don't pure components only re-render on updated props (so equivalent to `memo` with all props as deps)?

Reply:
pure components yeah - this is functional component (pure component is a class inheriting from `React.PureComponent` - functional components just re-render whenever called. `React.memo(functionalComponent)` makes the functional component equivalent to a `PureComponent`

Reply:
![cookie](https://user-images.githubusercontent.com/43223637/78204941-68125f00-74f7-11ea-8e21-047d800f5c64.gif)

Do you know any reasoning behind the design decision to not make functional components pure by default?

Reply:
nah - at a guess probably just that they were originally for 'smaller' things as classes were the default, so if they now make every functional component pure that would be a pretty massive backwards-incompatible change!